app/assets/javascripts/answers.js

```diff
@@ -20,6 +20,12 @@ $(document).ready(function(e){
   $("#answers").hide();
   $("#response").hide();
 
+  if ($('#answer_text').val() == "") {$('#submit_answer').attr('disabled', true);}
+
+  $('#answer_text').keyup(function(){
+    $('#submit_answer').attr('disabled', !($('#answer_text').val()));
```

Comment:
Ok great, now one more suggestion. The `if ...` and the code inside the `.keyup()` event function are actually doing exactly the same thing. But it's a little confusing for the reader / maintainer of the code, because they are written out in different ways. To tighten it up so it's even clearer and easier to maintain, why don't you refactor it so both places call a single function, and that function will then consistently perform the check against `#answer_text` and update `#submit_answer`. So something like this:
```
// enables submitting the form based on whether #answer_text has any input
function enableSubmitAnswer() {
  ...
}

// runs on page load
enableSubmitAnswer();

// runs each time the user presses a key in the #answer_text form field
('#answer_text').keyup(enableSubmitAnswer);
```